source/use-cases/sensitive-data-encryption.txt

```diff
@@ -124,7 +124,153 @@ In this step, we generate a local master key to expedite setup of our developmen
           }
 
 
-B. Define a JSON Schema
+B. Create a Data Encryption Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this section, we use the MongoDB driver to create and store a data
+encryption key, used by CSFLE to automatically encrypt and decrypt data on
+document fields. The following configuration values are required by the client
```

Comment:
nit: IMO the comma before "used" makes it read like an appositive. Instead we could do two sentences:

In this section we use the MongoDB driver to create and store a data encryption key _in the keyvault collection_. Drivers that have access to the keyvault can use the encryption key to encrypt and decrypt fields in a document.

Reply:
nit: passive -> active

The driver requires the following configuration values to generate a new data encryption key:

Reply:
Agree, it's awkward. I see that you want to emphasize/include discussion of the key vault collection. Since it's the first two sentences, would your goal be met if key vault collection was only mentioned in the second one?

Reply:
I think that's probably fine. The main reason I had it in both is to ensure that readers understand the keyvault is just a collection.